package.json

```diff
@@ -28,12 +28,17 @@
   "homepage": "https://github.com/BenevolentBactrians/Greenfield#readme",
   "dependencies": {
     "bcrypt": "^2.0.0",
+    "bcrypt-nodejs": "0.0.3",
     "body-parser": "^1.18.2",
+    "cookie-parser": "^1.4.3",
+    "ejs": "^2.5.9",
     "express": "^4.16.3",
```

Comment:
one question. why did you add all this modules? I don't see you using them("bcrypt-nodejs", "cookie-parser", "ejs")
Also in a future please do not add files not related to the pull request  topic. 
(EX. there are  spaces added or removed to
client/src/components/App.jsx ,
 client/src/index.jsx , 
database/index.js  and
server/index.js
 ) 
thanks